docs/operations/metrics.md

```diff
@@ -326,6 +326,12 @@ If `emitBalancingStats` is set to `true` in the Coordinator [dynamic configurati
 
 ## General Health
 
+### Overlord/Coordinator
+
+| Metric         | Description                                                                                                   | Dimensions      | Normal Value |
```

Comment:
Please use the formatting style used in the rest of the Druid docs.

Reply:
Updated. Intellij keep reformatting the doc, interesting.